docs/book/src/developer/providers/migrations/v1.10-to-v1.11.md

```diff
@@ -92,49 +277,78 @@ See [provider contracts](../contracts/overview.md) for more details.
   - `MachinePoolPhase`, used in `machinePool.status.phase`
   - `MachinePhase`, used in `machine.status.phase`
   
-### Removals
+## Removals
 
 There are no removals in the CAPI v1.11 version.
 
-As documented in [Suggested changes for providers](#suggested-changes-for-providers), please also start planning for future removals:
+However, as documented in [Suggested changes for providers](#suggested-changes-for-providers), it is highly reccomanded to start planning for future removals:
 
 - v1beta1 API version will be removed tentatively in August 2026
 - Starting from the CAPI release when v1beta1 removal will happen (tentative Aug 2026), the Cluster API project
-  will remove the Cluster API condition type, the `util/conditions` package, the code handling conditions in `util/patch.Helper`
-  and everything related to the custom Cluster API `v1beta.Condition` type.
+  will remove the Cluster API condition type, the `util/conditions/deprecated/v1beta1` package, 
+  the `util/deprecated/v1beta1` package package, the code handling conditions in `util/patch.Helper`
+  and everything related to the custom Cluster API custom condition type.
 - Compatibility support for the v1beta1 version of the Cluster API contract will be removed tentatively in August 2026
 
-### Other
+## Suggested changes for providers
+
+- We highly recommend providers to start using to Cluster API v1beta2 types when bumping to CAPI v1.11. 
+  This requires changing following import:
 
-### Suggested changes for providers
+  ```go
+  import (
+      ...
+      clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+  )
+  ```
+  
+  into:
+  
+  ```go
+  import (
+      ...
+      clusterv1 "sigs.k8s.io/cluster-api/api/v1beta2"
+  )
+  ```
+
+  Please refer to [API Changes](#api-changes) for more details about the changes introduced by this release.
+
+  Note: it is technically possible for providers to keep using v1beta1 types from CAPI v1.11, but this is 
+  not recommanded (this approach was not tested during the implementation, we don't know if there are road blockers). 
+
+- If providers are using Conditions from Cluster API resources, we highly recommend providers to check new 
```

Comment:
Can we dedup with the section below? (l.330+)

Reply:
The two sentences are about two different problems.

This sentence is about which condition you user e.g. when looking at the Cluster object
The paragraphs below is about what you do for conditions on provider's own resources

I tried to clarify, PTAL